docs/docs/blocks/!/TryMany.md

```diff
@@ -0,0 +1,42 @@
+# TryMany
+
+```clojure
+(TryMany
+  :Chain [(Chain) (String) (None) (ContextVar [(Chain)])]
+  :Policy [(Enum)]
+  :Threads [(Int)]
+  :Coroutines [(Int)]
+)
+```
+
+## Definition
+
+
+## Parameters
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| Chain | `[(Chain) (String) (None) (ContextVar [(Chain)])]` | `None` | The chain to spawn and try to run many times concurrently. |
+| Policy | `[(Enum)]` | `Enum: 1 vendor: 0x73696e6b type: 0x7472794d` | The execution policy in terms of chains success. |
```

Comment:
Another thing about Enum.. lisp side we should actually instead process them and print the real enum that is behind it.
I don't do it in the stringify cos it's expensive but maybe we should somehow.

Reply:
Maybe we can just postpone this?

Reply:
Yeah, we can have another issue/PR for that. The purpose of the current PR is to have the foundation for doc generation.